test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/actions/AuthActionSpec.scala

```diff
@@ -100,5 +100,32 @@ class AuthActionSpec extends ControllerSpec with MetricsMocks {
 
       redirectLocation(result) mustBe Some("/ppt-accounts-url")
     }
+
+    "redirect when user not logged in" in {
+
+      when(appConfig.loginUrl).thenReturn("login-url")
+      when(appConfig.loginContinueUrl).thenReturn("login-continue-url")
+
+      whenAuthFailsWith(MissingBearerToken())
+
+      val result =
+        createAuthAction().invokeBlock(authRequest(Headers(), PptTestData.newUser()),
+                                       okResponseGenerator
+        )
+
+      redirectLocation(result) mustBe Some("login-url?continue=login-continue-url")
+    }
+
+    "redirect when user not authorised" in {
```

Comment:
I think I'm right in saying this test covers the removed `handle insufficient enrolments authorisation exception` test? They look very similar but just want to confirm I'm correct in my assumption?

Reply:
Yes, basically.